Makefile

```diff
@@ -79,3 +79,9 @@ utils-docker:
 	rm -rf utils
 	mkdir -p utils
 	docker build -f docker-compose/utils/Dockerfile --output=utils .
+
+monitoring:
+	docker build -f docker-compose/monitoring/Dockerfile -t monitoring .
+	docker run \
+		-p 8080:8080 \
```

Comment:
this port is the same as for the LPS, since we're delivering in the same compose there is a change that they use this container in the same vm as the LPS so I'd suggest to use a different port

Reply:
Thanks. Changed.